### PR TITLE
Add -f -r flags to find-unused-errs

### PR DESCRIPTION
**Pull request by richsalz**
You can find only unused functions or error/reason codes, or both.

### CODE REVIEW

File: util/find-unused-errs [hunk 1]
```diff
@@ -13,8 +13,18 @@
 export X1=/tmp/f.1.$$
 export X2=/tmp/f.2.$$
 
+case "$1" in
+    -f) PAT='_F_'
+        echo Functions only
+        ;;
+    -[er]) PAT='_R_' ; echo Reason codes only
+        ;;
+    *) PAT='_[RF]_' ; echo Function and reason codes
```
Comment (dot-asm) [inline util/find-unused-errs]: Wouldn't it be more appropriate to return failure in case of arbitrary flag?

**Comment by richsalz** [comment]
updated commit pushed.

File: util/find-unused-errs [hunk 2]
```diff
@@ -13,8 +13,27 @@
...
+    -[er])
+        PAT='_R_'
+        echo Reason codes only
+        ;;
+    "")
+        PAT='_[FR]_'
+        echo Function and reasons
+        ;;
+    *)
+        echo "Usage error; one of -[efr] required."
```
Comment (levitte) [inline util/find-unused-errs]: That seems like an odd message...

Comment (richsalz) [inline util/find-unused-errs]: Enh.  It's a dev-team utility...

**Pull request closed without merging**